compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1220,6 +1220,34 @@ LogicalResult ArgCompareOp::reifyResultShapes(
       .reifyResultShapes(b, reifiedReturnShapes);
 }
 
+SmallVector<AffineMap>
+IREE::LinalgExt::ArgCompareOp::getIndexingMapsForOperands() {
+  Builder b(getContext());
+  const int64_t rank = getInputRank();
+  return SmallVector<AffineMap>{b.getMultiDimIdentityMap(rank)};
```

Comment:
nit: this is only used in one place, I'd inline it and use brace initialization
```suggestion
  return {b.getMultiDimIdentityMap(getInputRank())};
```